src/enums/avvisningsregelnavn.ts

```diff
@@ -0,0 +1,8 @@
+export const PASIENT_ELDRE_ENN_70 = 'PASIENT_ELDRE_ENN_70';
+export const UKJENT_DIAGNOSEKODETYPE = 'UKJENT_DIAGNOSEKODETYPE';
+export const UGYLDIG_KODEVERK_FOR_BIDIAGNOSE = 'UGYLDIG_KODEVERK_FOR_BIDIAGNOSE';
+export const UGYLDIG_REGELSETTVERSJON = 'UGYLDIG_REGELSETTVERSJON';
+export const BEHANDLER_IKKE_GYLDIG_I_HPR = 'BEHANDLER_IKKE_GYLDIG_I_HPR';
+export const BEHANDLER_MANGLER_AUTORISASJON_I_HPR = 'BEHANDLER_MANGLER_AUTORISASJON_I_HPR';
+export const BEHANDLER_IKKE_LE_KI_MT_TL_FT_I_HPR = 'BEHANDLER_IKKE_LE_KI_MT_TL_FT_I_HPR';
+export const BEHANDLER_SUSPENDERT = 'BEHANDLER_SUSPENDERT';
```

Comment:
dersom disse konstanten kun brukes i én (eller få filer) kan du vurdere å lage en enum i den aktuelle filen. I enkelte tilfeller synes jeg det kan være greit å samle ting som hører tett sammen i samme fil! Det er dog ikke sikkert det er riktig for denne situasjonen.

Reply:
Disse er ikke tatt i bruk enda, men jeg lager en TODO i trello på det. Dette er en del av branchen "avvist_sykmelding" som ble med her.

Slettes.